app/src/test/java/com/google/droidjump/ExampleUnitTest.java

```diff
@@ -16,10 +16,9 @@
 
 package com.google.droidjump;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
as above, add a new line between the static imports and the rest of the imports

Reply:
Fixed in all files.